v21.1/restore.md

```diff
@@ -88,19 +93,14 @@ When you restore a full cluster with an enterprise license, it will restore the
 
 #### Databases
 
-Restoring a database will create a new database and restore all of its tables and views.
-
-The created database will have the name of the database in the backup. The database cannot already exist in the target cluster.
-
-If [dropping](drop-database.html) or [renaming](rename-database.html) an existing database is not an option, you can use _table_ restore to restore all tables into the existing database:
+Restoring a database will create a new database and restore all of its tables and views. The created database will have the name of the database in the backup. The database cannot already exist in the target cluster.
```

Comment:
~Same comments as above re: I think we should say "it can't have the same name" first, before the other stuff.